subsys/bluetooth/mesh/model_utils.h

```diff
@@ -162,6 +162,14 @@ model_transition_is_active(const struct bt_mesh_model_transition *transition)
 	return (transition->time > 0 || transition->delay > 0);
 }
 
+static inline bool
+model_transition_is_invalid(const struct bt_mesh_model_transition *transition)
+{
+	return (transition != NULL &&
+		(transition->time > BT_MESH_MODEL_TRANSITION_TIME_MAX_MS ||
+		 transition->delay > BT_MESH_MODEL_DELAY_TIME_MAX_MS));
+}
```

Comment:
This causes `model_transition_is_invalid(NULL) == false`. To me the better semantics would be for NULL to be invalid?

Reply:
No, transition is not invalid if it is NULL. That is a valid input, as said in the description of the function.